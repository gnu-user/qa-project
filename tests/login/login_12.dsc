TITLE
    Privileged Login (admin)
DESCRIPTION
    Login using a admin account. Attempt to initiate all transactions. Verify 
    that each transaction does not produce an error.
DEPENDENCIES
