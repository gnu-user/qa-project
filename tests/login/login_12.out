Welcome.
> 
Enter username: 
Username accepted.
Available tickets file read successfully.
> 
Enter the username to create: 